selinux/ceph.te

```diff
@@ -84,8 +84,8 @@ logging_send_syslog_msg(ceph_t)
 sysnet_dns_name_resolve(ceph_t)
 
 # basis for future security review
-allow ceph_t ceph_var_run_t:sock_file { create unlink write };
-allow ceph_t self:capability sys_rawio;
+allow ceph_t ceph_var_run_t:sock_file { create unlink write setattr };
+allow ceph_t self:capability { sys_rawio chown };
```

Comment:
yeah, we do `chown_path()` in

- `global_init()` (setuid,setgid)
- `global_init_postfork_start()` (daemon) 
- `global_init_prefork()` (client) (client)